app/controllers/posts_controller.rb

```diff
@@ -1,20 +1,24 @@
 class PostsController < ApplicationController
   def new
+    # p "New: #{params}"
```

Comment:
If all we've done is add commented out debgug prints, I don't think we should include the file in the commit as we'll just have to delete those later anyway.